src/components/Settings/Notifications/NotificationsPushover.tsx

```diff
@@ -0,0 +1,257 @@
+import React from 'react';
```

Comment:
Let's put this file in a subfolder similar to how Slack is now. This will help separate the i18n keys since they are generated by the path names

Reply:
I see. I will fix it.